components/counter-bar/src/index.js

```diff
@@ -193,9 +197,15 @@ const CounterWrapper = styled(TotalWrapper, {
  *   <CounterBar.Total score={15}>All counters</CounterBar.Total>
  *   <CounterBar.Counters>
  *     <CounterBar.Counter score={1}>Counter 1</CounterBar.Counter>
- *     <CounterBar.Counter score={2} scoreColor="orange" scoreBackgroundColor="blue">Counter 2</CounterBar.Counter>
+ *     <CounterBar.Counter
+ *      score={2}
+ *      scoreColor="orange"
+ *      scoreBackgroundColor="blue">Counter 2</CounterBar.Counter>
```

Comment:
similar formatting comments apply here

and indeed through the rest of these examples